provisioning/image_customization.go

```diff
@@ -51,13 +57,41 @@ func setIronicBaseUrl(name string, info *ProvisioningInfo) corev1.EnvVar {
 	}
 }
 
+func setImagePublishUrl(imageCustomizationService string, imagesBindAddress string) corev1.EnvVar {
+	ips, err := net.LookupIP(imageCustomizationService)
+	if err == nil {
+		for _, ip := range ips {
+			if ip.IsLoopback() {
+				continue
+			}
+			if ip != nil {
+				if !strings.HasPrefix(imagesBindAddress, ":") {
+					imagesBindAddress = ":" + imagesBindAddress
+				}
```

Comment:
Why not just pass in a port number? It seems weird to call it an address when the hostname has to be blank.

Reply:
Not an issue in the latest implementation.